workflow_array_ephys/export.py

```diff
@@ -13,3 +11,21 @@
     "ecephys_session_to_nwb",
     "write_nwb",
 ]
+
+# Import ephys NWB export functions
+if ephys_mode == "no-curation":
+    from element_array_ephys.export.nwb import ecephys_session_to_nwb, write_nwb
+else:
+    missing_ephys_warning = (
+        f"Warning: ephys export requires the no-curation ephys_mode. To use it,\n\t"
+        + "try setting datajoint.config['custom']['ephys_mode'] to 'no-curation'\n\t"
+        + "and restarting your kernel."
+    )
+
+    def ecephys_session_to_nwb(*args):
+        return missing_ephys_warning
+
+    def write_nwb(*args):
+        return missing_ephys_warning
+
+    print(missing_ephys_warning)
```

Comment:
```suggestion
    raise NotImplementedError(
        "Ephys export requires the no-curation ephys_mode. To use it,\n\t"
        + "try setting datajoint.config['custom']['ephys_mode'] to 'no-curation'\n\t"
        + "and restarting your kernel."
    )
```

Reply:
I didn't want to raise an error here because that would halt the import process, and would prevent a user from trying out the upstream import functions.

My motivation for adding these dummy functions was to appease the linter. 
1. If a function isn't used, as is the case with this didactic script, the linter flags an issue. So, we add it to `__all__`
2. If a function isn't defined, but it is in `__all__`, as would be the case for an `acute` user, the import process throws an error. 

We can move forward with just accepting linter warnings for didactic scripts

Reply:
Thanks for clarifying.  You can ignore my suggestion.